src/index.html

```diff
@@ -22,12 +22,18 @@
       </header>
       <main class="content">
         <section class="profile page__section">
-          <img
-            class="profile__image"
-            src="<%=require('./images/jacques-cousteau.jpg')%>"
-            alt="avatar"
-          />
-          <button class="avatar__button" type="button"></button>
+          <div class="profile__avatar-container">
+            <img
+              class="profile__image"
+              src="<%=require('./images/jacques-cousteau.jpg')%>"
+              alt="avatar"
+            />
+            <button
+              class="avatar__button"

```

Comment:
`avatar__button`


There is a BEM mistake

`profile__avatar` can be used here as an element of `profile`